deploy/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+host_key_checking = False
```

Comment:
Do we really need this?
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#host-key-checking

Reply:
good point - removed in bb3903694bf7435aa57dd0c31c25e83197fae481